srcpkgs/mstflint/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'mstflint'
+pkgname=mstflint
+version=4.11.0.3
+revision=1
+wrksrc="${pkgname}-4.11.0"
+build_style=gnu-configure
+configure_args="--enable-fw-mgr --enable-openssl"
+makedepends="boost-devel libcurl-devel libibmad-devel liblzma-devel \
```

Comment:
You are inside a quoted variable, you don't need to use backslash to denote a new line